assets/questions/reverseWords.js

```diff
@@ -21,6 +21,8 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
   starterCode: {
     python:
 `def reverseWords(s):
+    # This reverseWords method processes a string (s)
```

Comment:
I think I mentioned this before, but we should add a period at the end.

Reply:
Period added.